localstack/services/apigateway/helpers.py

```diff
@@ -246,10 +246,25 @@ def get_resource_for_path(path, path_map):
         for match in matches:
             if match[0] == path:
                 return match
+            if test_path(path, match[0]):
+                return match
         raise Exception('Ambiguous API path %s - matches found: %s' % (path, matches))
     return matches[0]
 
 
+def test_path(path, apiPath):
```

Comment:
nitpick: Can we rename this function to something a little more descriptive - e.g., `def path_matches_pattern(..): ...`.

nitpick: We're usually using `snake_case` instead of `camelCase` - can we please rename to `api_path` (or `api_path_pattern`)?

Reply:
Yep will do